### DISCUSSION TITLE
How can I obtain git diff of the HEAD with the last made commit?

### DISCUSSION THREAD

**Discussion by IgnisDa** [ishepard/pydriller#165] [Q&A] [answered]
Basically, I want the same result as you'd get when you run `git diff` in the terminal.  Right now, `file.parsed_diff` gives the diff with the last commit, but I want it to be with the current state repository with the last made commit.

**Comment by ishepard** [comment, marked as answer]
Hi @IgnisDa!
You currently can't do it with Pydriller, as it can only traverse commits. Your best bet would be to use GitPython for this specific operation 👍
